content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -196,7 +196,7 @@ const GroupedActivityFeed = React.createClass({
               {sites.map((site, i) => {
                 globalCount++;
                 return (<ActivityFeedItem
-                    key={site.cacheKey || i}
+                    key={i}
```

Comment:
I filed https://github.com/mozilla/activity-stream/issues/712 to fix this situation once and for all
/cc @rlr